### PR TITLE
Added instruction to run locally

### PR DESCRIPTION
**Pull request by ErikOstlund**
Package: `lib-react-components`

Closes # NA

Describe your changes:
Updated README file to have instructions to run repo locally.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/lib-react-components/README.md [hunk 1]
```diff
@@ -24,13 +24,17 @@ ES6
 import { ZooFooter } from '@zooniverse/react-components';
 ```
 
+## Run
+
+`npm start` to run locally on (https://localhost:6006).
```
Comment (srallen) [inline packages/lib-react-components/README.md:29]:
```suggestion
`yarn start` to run the storybook locally on (https://localhost:6006).
```

File: packages/lib-react-components/README.md [hunk 2]
```diff
@@ -24,13 +24,17 @@ ES6
...
+
 ## Tests
 
 `npm test` to run mocha tests.
```
Comment (srallen) [inline packages/lib-react-components/README.md:33]:
```suggestion
`yarn test` to run mocha tests.
```

File: packages/lib-react-components/README.md (hunk 1 above)
Comment (ErikOstlund) [reply packages/lib-react-components/README.md]: Updated

File: packages/lib-react-components/README.md (hunk 2 above)
Comment (ErikOstlund) [reply packages/lib-react-components/README.md]: Updated

**Pull request merged**